tests/Unit/MissingDataFileTest.php

```diff
@@ -1,20 +1,22 @@
 <?php
 
+namespace DivineOmega\Countries\Tests;
+
 use DivineOmega\Countries\Countries;
 use PHPUnit\Framework\TestCase;
 
 final class MissingDataFileTest extends TestCase
 {
     private $dataFilePath = __DIR__.'/../../src/DataSources/mledoze/countries/dist/countries.json';
 
-    protected function setUp()
+    protected function setUp(): void
```

Comment:
To be compatible with `PHPUnit\Framework\TestCase::setUp`, it should add `:void` type hint.